tasks/agent.py

```diff
@@ -816,7 +831,7 @@ def version(
 
 
 @task
-def get_integrations_from_cache(ctx, python, bucket, branch, integrations_dir, target_dir, integrations, awscli="aws"):
+def get_integrations_from_cache(ctx, python, bucket, branch, integrations_dir, target_dir, integrations, awscli=None):
```

Comment:
Would it be best to remove the `awscli` parameter from the function Since we are no longer passing from `omnibus/config/software/datadog-agent-integrations-py(2|3).rb` ?

Reply:
works for me